Misc/python-config.sh.in

```diff
@@ -69,6 +74,10 @@ do
     esac
 done
 
+if [ "$PY_EMBED" = "1" ] ; then
```

Comment:
I think no quote is needed on this line, PY_EMBED is defined, not empty and does not contain any white space.

Reply:
Fixed.